.gitignore

```diff
@@ -15,6 +15,10 @@ pids
 *.seed
 *.pid.lock
 
+# IDE Folders
+.idea/
+.fleet/
```

Comment:
This isn't for 'support new flags' commit. For convenience these lines could be added, but on a separate commit.